modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2280,6 +2282,7 @@ export class Wallet implements IWallet {
         });
         break;
       case 'enabletoken':
+        params.recipients = [];
```

Comment:
recipients must be empty to do `enabletoken`

Reply:
i don't think we should force it - this is coin agnostic. instead, we should leave it so it is `params.recipients`.

However, i do see the problem you're running into...
in the `send` function, we always construct 1 recipient:
```
    const recipients: SendManyOptions['recipients'] = [
      {
        address: params.address,
        amount: params.amount,
        tokenName: params.tokenName,
      },
    ];
```

in this case, maybe we filter out recipients that have address === undefined?

e.g.
```
const recipients = params.recipients.filter((recipient) => recipient.address !== undefined))
```